### DISCUSSION TITLE
Get users who liked a post

### DISCUSSION THREAD

**Discussion by EduardSchwarzkopf** [kevinzg/facebook-scraper#395] [Q&A] [answered]
i'm not quite sure if i'm just overseeing this, but can you get all the users liked a post?

Thanks and Cheers

**Comment by neon-ninja** [comment, marked as answer]
Yes - they're called reactors. You can get all users who reacted with any of the 7 possible reaction types, including Like. Pass `options={"reactors": True}` to `get_posts`. Examples can be seen at https://github.com/kevinzg/facebook-scraper/pull/194
